src/App.tsx

```diff
@@ -1,25 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { Pagination } from './components/Pagination';
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [itemsPerPage, setItemsPerPage] = useState(5);
```

Comment:
since we have certain list of possible values i'd suggest you to create an array of them to avoid using so called [magic numbers](https://medium.com/@samaddico/programming-and-magic-numbers-f766e0cd1369)